src/tables.jl

```diff
@@ -214,14 +214,15 @@ function tableinfo(db::DB, name::AbstractString)
 end
 
 """
-    source |> SQLite.load!(db::SQLite.DB, tablename::String; temp::Bool=false, ifnotexists::Bool=false, replace::Bool=false, analyze::Bool=false)
-    SQLite.load!(source, db, tablename; temp=false, ifnotexists=false, replace::Bool=false, analyze::Bool=false)
+    source |> SQLite.load!(db::SQLite.DB, tablename::String; temp::Bool=false, ifnotexists::Bool=false, replace::Bool=false, or::Union{String, Nothing} = nothing, analyze::Bool=false)
```

Comment:
What is `or` here? Should this be `on_conflict` instead?

Reply:
Ah yes, seems I forgot to update that.